pkgs/by-name/en/ente-web/package.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  nodejs,
+  yarnConfigHook,
+  yarnBuildHook,
+  extraBuildEnv ? null,
+
+}:
+stdenv.mkDerivation (finalAttrs: rec {
```

The whole point of being able to pass a function to `stdenv.mkDerivation`, is to not use `rec`:

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

Use `finalAttrs.version` (for example) when needed - like you use `finalAttrs.src`.